docs/whats-new/dotnet-2020-05.md

```diff
@@ -23,7 +23,7 @@ Welcome to what's new in .NET docs for May 2020. This article lists some of the
 ### New articles
 
 - [CentOS 8 Package Manager - Install .NET Core](../core/install/linux-package-manager-centos8.md) - Add install instructions for CentOS 8
-- [Fedora 32 Package Manager - Install .NET Core](../core/install/linux-package-manager-fedora32.md) - Add installation instructions for Fedora 32
+- [Fedora 32 Package Manager - Install .NET Core](../core/install/linux-fedora.md) - Add installation instructions for Fedora 32
```

Comment:
```suggestion
- [Fedora Package Manager - Install .NET Core](../core/install/linux-fedora.md) - Add installation instructions for Fedora
```

Reply:
Since this is a what's new file, it wouldn't be accurate to link to a different article, so I'll just delete the link.